apps/web-giddh/src/app/project-wise-accounting/project-wise-accounting.service.ts

```
@@ -0,0 +1,227 @@
+    ) {
+    }
+
+    /**
+     * Replaces placeholders in a URL with corresponding values from a model object.
+     * @param url - The URL containing placeholders like `:key`.
+     * @param model - An object containing key-value pairs to replace in the URL.
+     * @returns The formatted URL with placeholders replaced.
+     */
+    public replaceUrlPlaceholders(url: string, model: Record<string, any>): string {
+        if (!url || !model) return url;
+        if (model.hasOwnProperty('companyUniqueName')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)